PICS/PICSDelegate.swift

```diff
@@ -27,8 +27,11 @@ class PICSDelegate: SpeziAppDelegate {
                     .requires(\.name),
 
                     // additional values stored using the `FirestoreAccountStorage` within our Standard implementation
+                    
+                    .requires(\.dateOfBirth),
                     .collects(\.genderIdentity),
-                    .collects(\.dateOfBirth)
+                    .supports(\.height),
```

Comment:
We might want to change this to collects instead of supports to collect the information during onboarding (though not as required)